src/main/java/io/kestra/plugin/redis/services/Options.java

```diff
@@ -0,0 +1,52 @@
+package io.kestra.plugin.redis.services;
+
+import io.lettuce.core.SetArgs;
+import lombok.Builder;
+import lombok.extern.jackson.Jacksonized;
+
+@Builder
+@Jacksonized
+public class Options {
```

Comment:
on this class, please add the doc from [here][https://redis.io/commands/set/)
to be more coherent with other task in Kestra, use long name also ex => expireTime, we are not a tcp protocol, don't need to limit readability for few bytes :smile: